src/main/java/com/bootcamp/demo/security/SecurityConfig.java

```diff
@@ -12,7 +12,7 @@
 
     @Override
     public void configure(HttpSecurity http) throws Exception {
-        http    .csrf().disable()
+        http    .csrf().disable().httpBasic().and()
```

Comment:
Please be aware that this config will enable HTTP Basic authentication for the application. We should keep the existing security configs - login and session cookie.